tools/pre-commit

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+# Copyright 2025 SmartThings, Inc.
+# Licensed under the Apache License, Version 2.0
+#
+# pre-commit
+#
+# - Soft link into .git/hooks/pre-commit
+#   - $ ln -s tools/pre-commit .git/hooks/pre-commit
```

Comment:
Maybe it's different with your version of `ln`, but usually the first argument is assigned to the symlink verbatim, rather than computed from where you invoked the command. So, it should be `../../tools/pre-commit`.